workflow-templates/docker-build.yaml

```
@@ -0,0 +1,38 @@
+name: DockerBuild
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  build-and-push-image:
+    name: Build And Push Image
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Set Variables
+        run: |
+          APPLICATION_NAME=${GITHUB_REPOSITORY,,}
```

```suggestion
          APPLICATION_NAME=${GITHUB_REPOSITORY,,} # convert repo name to lowercase
```

Since this can be used by more people, we should explain weird syntax just in case

---

maybe just stop using BASH?